config/locales/fr.yml

```diff
@@ -695,21 +695,21 @@ fr:
           update: 'Mise à jour'
         placeholders:
           optional: Optionel
-          about: 'Présentez-vous! Pourquoi vous êtes ici et quel but souhaitez-vous atteindre? Merci de prendre note que ceci sera visualisez par les utilisateurs qui vous recherche.'
+          about: 'Présentez-vous! Pourquoi vous êtes ici et quel but souhaitez-vous atteindre ? Merci de prendre note que ceci sera visualisez par les utilisateurs qui vous recherche.'
           required: Requis
         delete:
-          text: 'Est-ce que <strong>if me</strong> ne vous convient pas?'
+          text: 'Est-ce que <strong>if me</strong> ne vous convient pas ?'
           button: 'Supprimé mon compte de manière permanente'
-          confirm: 'Êtes-vous sûr? Vous allez perdre tous vos données. Mais vous pourrez toujours vous réenregistrer!'
+          confirm: 'Êtes-vous sûr ? Vous allez perdre tous vos données. Mais vous pourrez toujours vous réenregistrer !'
       new:
         subtitle: 'Commencez à raconter vos histoires et maintenez votre santé dans un espace privé et sécurisé'
         label: 'Quelques détails'
     sessions:
       subtitle: 'Bon retour parmi nous, nous nous réjouissons de pouvoir vous aider à attaquer cette journée'
     shared:
-      sign_in_google: 'S''identifier avec Google'
-      sign_up_google: 'S''enregistrer avec Google'
-      confirmation: 'Vous n''avez pas reçu vous informations de confirmation?'
-      unlock: 'Vous n''avez pas reçu vos informations pour débloquer votre compte?'
+      sign_in_google: "S'identifier avec Google"
```

Comment:
I think we should be consistent about whether we want to use double-quotes or single quotes. Your changes above got rid of the double quotes. But this change introduces them. I would pick single quotes.